src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Ball newBall = new Ball();
+        newBall.setColor(colorSupplier.getRandomColor());
```

Comment:
The `setColor` method is being passed a lowercase string from `colorSupplier.getRandomColor()`. If the `Ball` class is designed to work with `Color` types, consider modifying `ColorSupplier` to return a `Color` type instead of a string. This will ensure consistency and type safety across the application.